dev/benchmarks/complex_layout/windows/flutter/generated_plugin_registrant.cc

```diff
@@ -0,0 +1,11 @@
+//
```

Comment:
This file shouldn't be necessary for macOS tests. Similar for the other Windows files below.